sql-statements/sql-statement-show-placement-for.md

```diff
@@ -11,7 +11,13 @@ summary: The usage of SHOW PLACEMENT FOR in TiDB.
 >
 > If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
 
-`SHOW PLACEMENT FOR` summarizes all placement options from direct placement and placement policies, and presents them in the canonical form for a specific table, database schema, or partition.
+`SHOW PLACEMENT FOR` summarizes all placement options, and presents them in the canonical form for a specific table, database schema, or partition.
+
+The `Scheduling_State` indicates the current progress the Placement Driver (PD) has made in scheduling the placement:
+
+* `PENDING`: The PD has not yet started scheduling the placement. This might indicate that that the placement rules are semantically correct, but can not currently be satisfied by the cluster. For example, if `FOLLOWERS=4` but there are only 3 TiKV stores which are candidates for followers.
+* `INPROGRESS`: The PD is currently scheduling the placement.
+* `SCHEDULED`: The PD has successfully scheduled the placement.
```

Comment:
I guess these are column fields in the output of `SHOW PLACEMENT FOR`. Would it be better to place these descriptions in the Example section?

Reply:
I would prefer above, since it applies to all examples. But it's not a strong opinion.